### PR TITLE
Migrate from rgaudin debian image to kiwix maintenance image

### PR DESCRIPTION
**Pull request by benoit74**
Nota: this includes a migration from `docker.io/library/python:3.9-slim` to `docker.io/library/python:3.12-bookworm` ; vigilance will be needed when deploying to ensure we do not break something

### CODE REVIEW

**Comment by rgaudin** [comment]
I think we should deploy before merging.

**Comment by benoit74** [comment]
Yes, but wanted to have approval before deploying since I might have missed something ^^ I will deploy and monitor.

**Comment by benoit74** [comment]
Tested successfully on dev library (a minor fix was needed due to zimscraperlib update). I don't think testing other jobs is worth it (they are "just" manual maintenance jobs / deployments)

**Review by rgaudin — APPROVED** [review]
👍

**Pull request merged**
